index.html

```diff
@@ -11,42 +11,173 @@
+                </div>
+            </div>
+            <div class="card-7 card"><h2 class="Title-post">Profesional Art Printing Data</h2>
+                <p class="Supporting-text">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+                <div class="tags">
+                    <div class="tag-link">
+                        <span class="tag "><a href="https://www.html.org">html</a></span>
+                        <span class="tag "><a href="https://getbootstrap.com/">bootstrap</a></span>
+                        <span class="tag "><a href="https://www.ruby-lang.org/en/">Ruby</a></span>
+                    </div>
+                    <span class="see-project"><a href="">See project</a></span>
+                </div>
+                
+            </div>
+        </div> 
+        <div class="About">
+            <h2 class="about-me">About me</h2>
+            <span><p class="description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p></span>
+            <span class="get-resume"><a href="">Get My Resume</a></span>
+            <div class="image-2"></div>
+            <hr class="rule-2"><br>
+                <div class="offer">
+                    <ul class="language">
+                        <li class="heading list-1">Language</li>
+                        <ul class="list">
+                            <li><a href="">Javascript</a></li>
+                            <li><a href="">Ruby</a></li>
+                            <li><a href="">Html</a></li>
+                            <li><a href="">Css</a></li>
+                        </ul>
                     </ul>
-                </div> 
-               
+                    <ul class="frameworks">
+                        <li class="heading list-2">Frameworks</li>
+                        <ul class="list">
+                            <li><a href="">Bootstrap</a></li>
+                            <li><a href="">Ruby on Rails</a></li>
+                            <li><a href="">RSPec</a></li>
+                            <li><a href="">CapyBara</a></li>
+                            <li><a href="">Codekit</a></li>
+                        </ul>
+                    </ul>
+                    <ul class="skills">
+                        <li class="heading list-3">Skill</li>
+                        <ul class="list">
+                            <li><a href="">Codekit</a></li>
+                            <li><a href="">Github</a></li>
+                            <li><a href="">Codepen</a></li>
+                            <li><a href="">Gitlab</a></li>
+                            <li><a href="">Terminal</a></li>
+                        </ul>
+                    </ul>
+                </div>
```

Comment:
- [ ] Your design doesn't look aligned and there are also some spacing issues that need to be fixed. Please refer to the images below to correct these as it is making your design look different from the Figma design. Please fix this as well. 
- Your design
![image](https://user-images.githubusercontent.com/102699958/212469487-13a83c47-f360-49a2-aeae-cc12c769ed1f.png)
- The Figma Design
![image](https://user-images.githubusercontent.com/102699958/212469507-192d4c34-1b97-43ca-a56a-31e751b23564.png)

Reply:
I love 🤩  what you did here with the bullets but make sure to keep your design centered and perhaps, keep all the bullets the same size 😉 , you also seem to be having some overflowing issues all along the page. Make sure to handle that to improve the User Experience. Keep going, you rock 💪🏽 

## Your design on `375px` as the witdh
![Screenshot 2023-01-14 at 10 57 58](https://user-images.githubusercontent.com/45739963/212481356-601efd01-041a-452e-96cf-eaa75bd58f22.png)